src/lib.rs

```diff
@@ -104,10 +104,11 @@
 //!
 //! ## State
 //!
-//! Middleware often needs to share values with the endpoint. This is done through "local state".
-//! Local state is built using a typemap that's available through [`Request::local`].
+//! Middleware often needs to share values with the endpoint. This is done through "request scoped
+//! state".  Request scoped state is built using a typemap that's available through
+//! [`Request::ext`].
```

Comment:
Should also mention `Response::ext`

Reply:
I added a simple paragraph for Response extensions. Thanks.